src/js/components/Box/doc.js

```diff
@@ -129,6 +129,7 @@ export const doc = Box => {
           PropTypes.oneOf(['xsmall', 'small', 'medium', 'large', 'xlarge']),
           PropTypes.string,
         ]),
+        style: PropTypes.string,
```

Comment:
how about `shape` instead of `style`?

Also, can we list the PropTypes instead of just using string ?

`PropTypes.oneOf(['solid', 'dashed', ..]).defaultValue('solid')`

Reply:
by speaking with @ericsoderberghp let's keep it `style` since the css variable is `border-style`

Reply:
and done :)